src/serialize.h

```diff
@@ -1146,4 +1020,16 @@ inline void SerReadWriteMany(Stream& s, int nType, int nVersion, CSerActionUnser
     ::UnserializeMany(s, nType, nVersion, args...);
 }
 
+template <typename T>
+size_t GetSerializeSize(const T& t, int nType, int nVersion = 0)
```

Comment:
Is there a reason to have the default value of 0 for `nVersion`?
Seems worth thinking of especially with all our transaction versions (though I don't remember if we are using `nVersion` for those)

Reply:
A side benefit of reviewing this is that I finally understood what that weird `ADD_SERIALIZE_METHODS` macro is!

Reply:
`GetSerializeSize` is not just for transactions; it is used by *all* serialization operations in `zcashd`. The justification for leaving the default as 0 is that changing it could likely break a bunch of other parsers.

Reply:
Ahh, yeah now I see a default of 0 in all the old versions of the method.